lib/touch-enroll.js

```diff
@@ -0,0 +1,95 @@
+import { exec as nodeExec } from 'child_process';
+import B from 'bluebird';
+
+// Had problems with teen_process, resorted to this
+const exec = B.promisify(nodeExec);
+
+let touchEnrollMenuKeys = ['Toggle Enrolled State', 'Touch ID Enrolled'];
+let touchEnrollBackups;
```

Comment:
this is global to the file; what if multiple tests are running at once? should this be a class if we need to maintain state? or we could maintain state on the ios sim object?